Ports/nesalizer/patches/0001-Add-Serenity-to-Makefile.patch

```diff
@@ -0,0 +1,50 @@
+From 6ab34657eb919049cb57a09fd55e034df1850835 Mon Sep 17 00:00:00 2001
+From: Dan MacDonald <allcoms@gmail.com>
+Date: Sun, 8 Dec 2019 17:30:52 +0000
+Subject: [PATCH 1/2] Add Serenity to Makefile
+
+- Add `-lSDL2 -lgui -lipc -lgfx -lcore -lpthread -lregex`
+- Disable RTTI
+- Install executables to `/bin`
+---
+ Makefile | 9 ++++++---
+ 1 file changed, 6 insertions(+), 3 deletions(-)
+
+diff --git a/Makefile b/Makefile
+index 527a3d0..94a4e42 100644
+--- a/Makefile
++++ b/Makefile
+@@ -58,7 +58,7 @@ c_objects   = $(addprefix $(BUILD_DIR)/,$(c_sources:=.o))
+ objects     = $(c_objects) $(cpp_objects)
+ deps        = $(addprefix $(BUILD_DIR)/,$(c_sources:=.d) $(cpp_sources:=.d))
+ 
+-LDLIBS := $(shell sdl2-config --libs) -lrt
++LDLIBS := -lSDL2 -lgui -lipc -lgfx -lcore -lpthread -lregex
+ 
+ ifeq ($(INCLUDE_DEBUGGER),1)
+     LDLIBS += -lreadline
+@@ -105,12 +105,12 @@ else ifneq ($(MAKECMDGOALS),clean)
+ endif
+ 
+ ifneq ($(findstring debug,$(CONF)),)
+-    compile_flags += -ggdb
++    compile_flags += -ggdb -fno-rtti
+ endif
+ ifneq ($(findstring release,$(CONF)),)
+     # Including -Ofast when linking (by including $(optimizations)) gives a
+     # different binary size. Might be worth investigating why.
+-    compile_flags += $(optimizations) -DOPTIMIZING
++    compile_flags += $(optimizations) -DOPTIMIZING -fno-rtti
+     link_flags    += $(optimizations) -fuse-linker-plugin
+ endif
+ 
+@@ -175,3 +175,6 @@ $(objects) $(deps): | $(BUILD_DIR)
+ 
+ .PHONY: clean
+ clean: ; $(q)-rm -rf $(BUILD_DIR)
++
++install:
++	install $(BUILD_DIR)/${EXECUTABLE} ${DESTDIR}/bin/${EXECUTABLE}
```

Comment:
We should install to `/usr/local/bin` instead. In fact, it's probably a lot clearer to omit this patch chunk and just replace the `install` step in `package.sh` with a manual copy to the correct location.

Reply:
Thank you, I just copy the executable to `/usr/local/bin` now.